test/hotspot/jtreg/ProblemList.txt

```diff
@@ -118,7 +118,7 @@ serviceability/sa/ClhsdbFindPC.java#id1 8269982 macosx-aarch64
 serviceability/sa/ClhsdbFindPC.java#id3 8269982 macosx-aarch64
 serviceability/sa/ClhsdbPstack.java#id1 8269982 macosx-aarch64
 
-serviceability/dcmd/gc/HeapDumpAllTest.java 8270341 generic-x64
+#serviceability/dcmd/gc/HeapDumpAllTest.java 8270341 generic-x64
```

Comment:
Please delete the entry rather than commenting it out.

Reply:
Sorry. I initially commented it out for testing since I didn't plan on being the one to push this change, and forgot to delete it. Will fix.